bicycleparameters/main.py

```diff
@@ -1028,6 +1295,114 @@ def plot_eigenvalues_vs_speed(self, speeds, fig=None, generic=False,
             fig.show()
 
         return fig
+    
+    
+    def plot_eigenvalues_vs_speed_plotly(self, speeds, fig=None, generic=False,
+                                   color='black', show=False, largest=False,
+                                   linestyle='-', grid=False, show_legend=True):
```

Comment:
needs a docstring, also same comments about if all these optiosn work in plotly or not